src/commands/send.rs

```diff
@@ -149,7 +153,10 @@ impl FromStr for SupportedResponse {
         match input {
             "icrc1_balance_of" => Ok(SupportedResponse::AccountBalance),
             "icrc1_transfer" => Ok(SupportedResponse::Transfer),
+            "list_deployed_snses" => Ok(SupportedResponse::ListDeployedSnses),
             "manage_neuron" => Ok(SupportedResponse::ManageNeuron),
+            "send_dfx" => Ok(SupportedResponse::IcpTransfer),
```

Comment:
why is this called "send_dfx" if it's IcpTransfer?

Reply:
Legacy reasons that I didn't want to mess with in this particular MR. This is the method that is supported in the current checked in icp_ledger.did and used to transfer ICP around. I'd like to use the ic monorepo `transfer` as that's more up to date but `send_dfx` is still supported and what Adam included when writing the Swap command. I didn't want to take on too much in the PR. As for the naming choices: the `send_dfx` is what is matched on, but I thought a better name for `SupportedResponse` would be `IcpTransfer`.

Reply:
Sounds good.  If it's legacy, that's enough reason to leave it alone for now.  But agree it'd be nice to fix later